pupil_src/shared_modules/calibration_routines/gaze_mappers.py

```diff
@@ -76,68 +76,128 @@ class Binocular_Gaze_Mapper_Base(Gaze_Mapping_Plugin):
 
     def __init__(self, g_pool):
         super().__init__(g_pool)
+        self._cfg = {'min_pupil_confidence': 0.6,
+                     'temporal_cutoff': 0.03,
+                     'timestamp_reset_threshold': 5.0,
+                     'process_low_confidence': True,
+                     'sample_cutoff': 10}
```

Comment:
What is the reason for storing the configuration in `_cfg` instead of the class itself?

Reply:
As far as I can tell, that's what the GUI expects. These variables are all exposed and can be modified via the GUI.